plugins/easyimage/plugin.js

```diff
@@ -453,4 +556,80 @@
 	 * @member CKEDITOR.config
 	 */
 	CKEDITOR.config.easyimage_sideClass = 'easyimage-side';
+
+	/**
+	 * A CSS class representing image aligned to left.
+	 *
+	 *		// Changes the class to "my-left-image".
+	 *		config.easyimage_alignLeftClass = 'my-left-image';
+	 *
+	 * @since 4.9.0
+	 * @cfg {String} [easyimage_alignLeftClass='easyimage-alignLeft']
+	 * @member CKEDITOR.config
+	 */
+	CKEDITOR.config.easyimage_alignLeftClass = 'easyimage-alignLeft';
+
+	/**
+	 * A CSS class representing image aligned to center.
+	 *
+	 *		// Changes the class to "my-center-image".
+	 *		config.easyimage_alignCenterClass = 'my-center-image';
+	 *
+	 * @since 4.9.0
+	 * @cfg {String} [easyimage_alignCenterClass='easyimage-alignCenter']
+	 * @member CKEDITOR.config
+	 */
+	CKEDITOR.config.easyimage_alignCenterClass = 'easyimage-alignCenter';
+
+	/**
+	 * A CSS class representing image aligned to right.
+	 *
+	 *		// Changes the class to "my-right-image".
+	 *		config.easyimage_alignLeftClass = 'my-right-image';
+	 *
+	 * @since 4.9.0
+	 * @cfg {String} [easyimage_alignRightClass='easyimage-alignRight']
+	 * @member CKEDITOR.config
+	 */
+	CKEDITOR.config.easyimage_alignRightClass = 'easyimage-alignRight';
+
+	/**
+	 * Custom styles that could be applied to Easy Image widget. All styles must be instances of
+	 * {@link CKEDITOR.style}. There are three additional properties for every style:
+	 *
+	 * * `label` - string used as a button label in a balloon toolbar for the widget,
+	 * * `icon` - path to the icon used in the balloon toolbar,
+	 * * `iconHiDpi` - path to the high DPI version of the icon.
+	 *
+	 *		config.easyimage_styles = {
+	 *			left: {
+	 *				attributes: {
+	 *					'class': 'left'
+	 *				},
+	 *				label: 'Align left',
+	 *				icon: '/foo/bar/icons/baz.png',
+	 *				iconHidpi: '/foo/bar/icons/hidpi/baz.png'
+	 *			}
+	 *		};
+	 *
+	 * Every style added by this config variable will result in adding `Easyimage<name>` button
+	 * and `easyimage<name>` command, where `<name>` is name of style in pascal case, e.g. `left`
+	 * style would produce `EasyimageLeft` button and `easyimageLeft` command.
+	 *
+	 * @since 4.9.0
+	 * @cfg {Object<string,Object>} easyimage_styles
+	 * @member CKEDITOR.config
+	 */
+	CKEDITOR.config.easyimage_styles = {};
+
+	/**
+	 * List of buttons to be displayed in a balloon toolbar for Easy Image widget.
+	 * If Context Menu plugin is enabled, this config variable will be used also to add
+	 * items to the context menu for Easy Image widget.
+	 *
+	 * @since 4.9.0
+	 * @cfg {Array/String} [easyimage_toolbar=[ 'EasyimageFull', 'EasyimageSide', 'EasyimageAlt' ]]
```

Comment:
It's expected to contain an array of strings, so accurate type hint is `String[]/String`.